ERROR - 2021-05-11 14:08:06 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2021-05-11 14:08:06 --> Unable to connect to the database.
#0 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\cs\app\Models\Querys.php(32): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\cs\app\Helpers\SanPedro_helper.php(10): App\Models\Querys->verifyUser(Array)
#4 C:\xampp\htdocs\cs\app\Controllers\Auth.php(32): authenticated()
#5 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Auth->login()
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2021-05-11 14:09:02 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2021-05-11 14:09:02 --> Unable to connect to the database.
#0 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\cs\app\Models\Querys.php(32): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\cs\app\Helpers\SanPedro_helper.php(10): App\Models\Querys->verifyUser(Array)
#4 C:\xampp\htdocs\cs\app\Controllers\Auth.php(32): authenticated()
#5 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Auth->login()
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2021-05-11 14:56:52 --> mysqli_sql_exception: Unknown column 'tomando_medicamento' in 'field list' in C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php(314): mysqli->query('UPDATE `sp_trat...')
#1 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sp_trat...')
#2 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sp_trat...')
#3 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(2468): CodeIgniter\Database\BaseConnection->query('UPDATE `sp_trat...', Array, false)
#4 C:\xampp\htdocs\cs\app\Models\EnfermedadModel.php(36): CodeIgniter\Database\BaseBuilder->update(Array, Array)
#5 C:\xampp\htdocs\cs\app\Controllers\Enfermedad.php(206): App\Models\EnfermedadModel->enfermedad('update', Array, Array, NULL)
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Enfermedad->guardar_enfermedad()
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enfermedad))
#8 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-05-11 14:56:52 --> Unknown column 'tomando_medicamento' in 'field list'
#0 C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php(314): mysqli->query('UPDATE `sp_trat...')
#1 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sp_trat...')
#2 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sp_trat...')
#3 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(2468): CodeIgniter\Database\BaseConnection->query('UPDATE `sp_trat...', Array, false)
#4 C:\xampp\htdocs\cs\app\Models\EnfermedadModel.php(36): CodeIgniter\Database\BaseBuilder->update(Array, Array)
#5 C:\xampp\htdocs\cs\app\Controllers\Enfermedad.php(206): App\Models\EnfermedadModel->enfermedad('update', Array, Array, NULL)
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Enfermedad->guardar_enfermedad()
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enfermedad))
#8 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2021-05-11 14:57:16 --> mysqli_sql_exception: Unknown column 'tomando_medicamento' in 'field list' in C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php(314): mysqli->query('UPDATE `sp_trat...')
#1 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sp_trat...')
#2 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sp_trat...')
#3 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(2468): CodeIgniter\Database\BaseConnection->query('UPDATE `sp_trat...', Array, false)
#4 C:\xampp\htdocs\cs\app\Models\EnfermedadModel.php(36): CodeIgniter\Database\BaseBuilder->update(Array, Array)
#5 C:\xampp\htdocs\cs\app\Controllers\Enfermedad.php(206): App\Models\EnfermedadModel->enfermedad('update', Array, Array, NULL)
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Enfermedad->guardar_enfermedad()
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enfermedad))
#8 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-05-11 14:57:16 --> Unknown column 'tomando_medicamento' in 'field list'
#0 C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php(314): mysqli->query('UPDATE `sp_trat...')
#1 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sp_trat...')
#2 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sp_trat...')
#3 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(2468): CodeIgniter\Database\BaseConnection->query('UPDATE `sp_trat...', Array, false)
#4 C:\xampp\htdocs\cs\app\Models\EnfermedadModel.php(36): CodeIgniter\Database\BaseBuilder->update(Array, Array)
#5 C:\xampp\htdocs\cs\app\Controllers\Enfermedad.php(206): App\Models\EnfermedadModel->enfermedad('update', Array, Array, NULL)
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Enfermedad->guardar_enfermedad()
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enfermedad))
#8 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2021-05-11 15:01:46 --> mysqli_sql_exception: Unknown column 'tomando_medicamento' in 'field list' in C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php(314): mysqli->query('UPDATE `sp_trat...')
#1 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sp_trat...')
#2 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sp_trat...')
#3 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(2468): CodeIgniter\Database\BaseConnection->query('UPDATE `sp_trat...', Array, false)
#4 C:\xampp\htdocs\cs\app\Models\EnfermedadModel.php(36): CodeIgniter\Database\BaseBuilder->update(Array, Array)
#5 C:\xampp\htdocs\cs\app\Controllers\Enfermedad.php(208): App\Models\EnfermedadModel->enfermedad('update', Array, Array, NULL)
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Enfermedad->guardar_enfermedad()
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enfermedad))
#8 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-05-11 15:01:46 --> Unknown column 'tomando_medicamento' in 'field list'
#0 C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php(314): mysqli->query('UPDATE `sp_trat...')
#1 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sp_trat...')
#2 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sp_trat...')
#3 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(2468): CodeIgniter\Database\BaseConnection->query('UPDATE `sp_trat...', Array, false)
#4 C:\xampp\htdocs\cs\app\Models\EnfermedadModel.php(36): CodeIgniter\Database\BaseBuilder->update(Array, Array)
#5 C:\xampp\htdocs\cs\app\Controllers\Enfermedad.php(208): App\Models\EnfermedadModel->enfermedad('update', Array, Array, NULL)
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Enfermedad->guardar_enfermedad()
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enfermedad))
#8 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2021-05-11 15:01:58 --> mysqli_sql_exception: Unknown column 'tomando_medicamento' in 'field list' in C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php(314): mysqli->query('UPDATE `sp_trat...')
#1 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sp_trat...')
#2 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sp_trat...')
#3 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(2468): CodeIgniter\Database\BaseConnection->query('UPDATE `sp_trat...', Array, false)
#4 C:\xampp\htdocs\cs\app\Models\EnfermedadModel.php(36): CodeIgniter\Database\BaseBuilder->update(Array, Array)
#5 C:\xampp\htdocs\cs\app\Controllers\Enfermedad.php(208): App\Models\EnfermedadModel->enfermedad('update', Array, Array, NULL)
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Enfermedad->guardar_enfermedad()
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enfermedad))
#8 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-05-11 15:01:58 --> Unknown column 'tomando_medicamento' in 'field list'
#0 C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php(314): mysqli->query('UPDATE `sp_trat...')
#1 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sp_trat...')
#2 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sp_trat...')
#3 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(2468): CodeIgniter\Database\BaseConnection->query('UPDATE `sp_trat...', Array, false)
#4 C:\xampp\htdocs\cs\app\Models\EnfermedadModel.php(36): CodeIgniter\Database\BaseBuilder->update(Array, Array)
#5 C:\xampp\htdocs\cs\app\Controllers\Enfermedad.php(208): App\Models\EnfermedadModel->enfermedad('update', Array, Array, NULL)
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Enfermedad->guardar_enfermedad()
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enfermedad))
#8 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-05-11 21:25:22 --> syntax error, unexpected '?>', expecting ')'
#0 C:\xampp\htdocs\cs\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp\htdocs\cs\system\Common.php(1217): CodeIgniter\View\View->render('inicio/nuestros...', Array, true)
#2 C:\xampp\htdocs\cs\app\Controllers\Inicio.php(21): view('inicio/nuestros...')
#3 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Inicio->nuestrosDoctores()
#4 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inicio))
#5 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#7 {main}
